### PR TITLE
Changes after evaluating/tuning fire model

### PR DESCRIPTION
**Pull request by orianac**

### CODE REVIEW

File: carbonplan_forests/plot/fire.py [hunk 1]
```diff
@@ -126,3 +181,57 @@ def evaluation(data, model, data_var='vlf', model_var='prob', projection='albers
     )
 
     return chart
+
+
+def package_for_altair(fire, climate, label='value'):
```
Comment (freeman-lab) [inline carbonplan_forests/plot/fire.py:186]: i don't see this function used anywhere, maybe drop from the PR...

File: notebooks/climate/climate_analysis.ipynb [hunk 2]
```diff
@@ -0,0 +1,561 @@
+{
```
Comment (freeman-lab) [inline notebooks/climate/climate_analysis.ipynb:1]: is this notebook part of the fire analysis updates? doesn't seem to be?

File: carbonplan_forests/prepare.py [hunk 3]
```diff
@@ -54,21 +60,34 @@ def fire(climate, nftd, mtbs=None, eval_only=False, scramble=False):
...
-                        for a in climate['tmean'].groupby('time.year').max()
+                        for a in climate['tmean'].rolling(dim={'time': 12}, center=False).max()
                     ]
                 ).flatten(),
                 np.asarray(
-                    [
+                    [  # and i want to do the same here but using sum instead of max (which i've already changed)
                         np.tile(a.mean(), [12, shape[1], shape[2]])
-                        for a in climate['ppt'].groupby('time.year').max()
+                        for a in climate['ppt'].groupby('time.year').sum()
```
Comment (freeman-lab) [inline carbonplan_forests/prepare.py:71]: should this be rolling as well?

File: carbonplan_forests/plot/fire.py [hunk 4]
```diff
@@ -126,3 +181,57 @@ def evaluation(data, model, data_var='vlf', model_var='prob', projection='albers
...
+        data,
+        id_vars=['year', 'month'],
+        value_vars=['fire (mtbs)', 'ppt', 'cwd', 'pdsi'],
+        value_name=label,
+        ignore_index=False,
+    )
+    return data
+
+
+def multipanel_slider(data, year_limits, region_labels):
```
Comment (freeman-lab) [inline carbonplan_forests/plot/fire.py:211]: maybe also drop this as part of this PR

File: carbonplan_forests/plot/fire.py (hunk 1 above)
Comment (orianac) [reply carbonplan_forests/plot/fire.py:186]: So, if I used these functions for analyses that didn't end up making it anywhere, but maybe I want to save them for future use- what's the best practice there? Just leave it on a miscellaneous branch? I guess it's stored in the commit history. I'll delete them for now.

File: notebooks/climate/climate_analysis.ipynb (hunk 2 above)
Comment (orianac) [reply notebooks/climate/climate_analysis.ipynb:1]: excellent point- will remove.

**Pull request merged**